data.py

```diff
@@ -25,6 +26,21 @@
 
 from src import schemas
 
+def process_json_files(g, dir_path, context_file):
```

Comment:
right idea but you cant pass context file in, it only exists in the scope of the "set_context" function.

its not the most efficient, but you can just do `g += Graph().parse(data=json.dumps(set*context(resource_dict)), format="json-ld")` at the end of this for now.